Makefile

```diff
@@ -61,10 +65,21 @@ localintegration: binaries
 
 binaries: ${OCID_LINK} ocid ocic conmon
 
-install:
+MANPAGES_MD = $(wildcard docs/*.md)
+
+docs/%.1: docs/%.1.md
+	$(GO_MD2MAN) -in $< -out $@.tmp && touch $@.tmp && mv $@.tmp $@
+
+docs/%.8: docs/%.8.md
+	$(GO_MD2MAN) -in $< -out $@.tmp && touch $@.tmp && mv $@.tmp $@
+
+docs: $(MANPAGES_MD:%.md=%)
+
+install: binaries docs
 	install -D -m 755 ocid ${INSTALLDIR}/ocid
 	install -D -m 755 ocic ${INSTALLDIR}/ocic
 	install -D -m 755 conmon/conmon ${INSTALLDIR}/conmon
+	install -D -m 644 $(basename $(MANPAGES_MD)) $(PREFIX)/share/man/man8
```

Comment:
I figured out a way to remove them on uninstall.  Just wanted to make sure if we add multiple man pages that it would work.